repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql

```diff
@@ -0,0 +1,6 @@
+DROP INDEX idx_reference  CASCADE;
+DROP INDEX idx_event__case_type_id  CASCADE;
```

Comment:
Please double-check the name of this file; it doesn't seem to match the convention:
`V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql` should be
`V20210716_436__CCD-436_Drop_duplicate_indexes.sql`

Reply:
Update/refactor File name to match convention